_pages/material.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: page
+title: "Apache Beam Material"
+permalink: /material/
+---
+
+This page contains project material for the Apache Beam project.
+
+## Project logos
+You can download [this archive]({{ site.baseurl }}/{{ site.downloads }}/{{ site.data.logos.archive-file }})
+containing all of the logos or download the logos individually.
+
+### Scalable Vector Graphics (SVG)
+These [SVG files](https://en.wikipedia.org/wiki/Scalable_Vector_Graphics) can
+be resized easily and are suitable for print or web use. Click on the logo to
+open for download.
```

Comment:
Well, minor nit but in Chrome, it opens the file which means you have to download it separately. :)